src/components/CardSingle.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+  /************************** 
+  Top level Card container
+  todo: fill out this section
+  ***************************/
+
+  import CardFooter from './CardFooter.vue'
+  
+  defineProps<{
+    card_title: string
+    card_description: string
+    card_url_main: string
```

Comment:
is the idea that choc sauce enriches this var?

Reply:
I would think so. Or feeds have a default values that could be overwritten when the component is used.